pages/login.vue

```diff
@@ -0,0 +1,5 @@
+<script setup lang="ts">
+definePageMeta({
+	middleware: () => navigateTo(getLoginURL(), { external: true, redirectCode: 301, replace: true })
```

Comment:
pageMeta can define a redirect. Can we not use that? https://nuxt.com/docs/api/utils/define-page-meta#type

Reply:
I check that, but seemed to support only local paths, not external ones. Also lacks of dynamic redirect support. This middleware has similar behaviour and does exactly what we want without even trying to render anything.